auto_changelog/domain_model.py

```diff
@@ -88,7 +88,12 @@ def add_release(self, *args, **kwargs):
 
     def add_note(self, *args, **kwargs):
         """ Add new Note to current release. Require same arguments as :class:`Note` """
-        note = Note(*args, **kwargs)
+        try:
+            note = Note(*args, **kwargs)
+        except ValueError:
+            # Ignore exceptions raised by unsupported commit type.
+            return
```

Comment:
Should we at least notify user about ignoring via log? What's your opinion @WqyJh?